src/components/search/Search.js

```diff
@@ -1,18 +1,68 @@
 'use strict';
 import React, {Component} from 'react';
 import {connect} from 'react-redux';
+import Autosuggest from 'react-autosuggest';
 import * as searchActions from './../../actions/search';
+import Spotify from './../../core/Spotify';
 
 class Search extends Component {
+    _handleKeyPress(event) {
+        if (event.key === 'Enter') {
+            const text = event.target.value;
+            if (text.length > 3) {
+                this._search(text);
+            }
+        }
+    }
```

Comment:
Why was _handleKeyPress moved from below render() to above? Not sure if it is a best practice but I think I've read a suggestion about custom methods should be put below native ones.